namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-submit-information-report-prod/resources/s3.tf

```diff
@@ -0,0 +1,30 @@
+module "hmpps_submit_information_report_s3_bucket" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=4.3"
+
+  team_name              = var.team_name
+  business-unit          = "Digital and Technology"
```

Comment:
Business unit is HMPPS everywhere apart from here. We intend to validate this value eventually so this should on this [list](https://ministryofjustice.github.io/technical-guidance/standards/documenting-infrastructure-owners/#mandatory).

Reply:
I have updated this